app/projects/skilled-elearning-landing-page/styles.js

```diff
@@ -0,0 +1,75 @@
+'use client';
+
+import styled from 'styled-components';
+
+export const theme = {
+  light: '#FFFFFF',
+  dark: '#13183F',
+  grey: '#83869A',
+  lightBlue: '#666CA3',
+  pink: '#F74780',
+  lightPink: '#FFA7C3',
+  gradient1: 'linear-gradient(180deg, #FF6F48 0.04%, #F02AA6 100%);',
+  gradient1hover:
+    'linear-gradient(0deg, rgba(255, 255, 255, 0.50) 0%, rgba(255, 255, 255, 0.50) 100%), linear-gradient(180deg, #FF6F48 0%, #F02AA6 100%);',
+  gradient2: 'linear-gradient(180deg, #4851FF 0%, #F02AA6 99.92%);',
+  gradient2hover:
+    'linear-gradient(0deg, rgba(255, 255, 255, 0.50) 0%, rgba(255, 255, 255, 0.50) 100%), linear-gradient(180deg, #4851FF 0%, #F02AA6 100%);',
+
+  gradient3: 'linear-gradient(180deg, #FFF 0%, #F0F1FF 100%)',
+};
+
+export const Wrapper = styled.section`
+  width: 100%;
+  max-width: 2100px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: ${theme.gradient3};
+  padding: 0rem 2rem;
+
+  ::selection {
+    background: ${theme.pink};
+    color: ${theme.light};
+  }
+
+  & .container {
+    width: 100%;
+    max-width: 1110px;
+
+    @media only screen and (min-width: 765px) and (max-width: 1279px) {
+      max-width: 768px;
+    }
+
+    @media (max-width: 764px) {
+      max-width: 343px;
+    }
```

Comment:
This is the file where you need to add the Container component and export from here